src/registrar/tests/test_views.py

```diff
@@ -1372,6 +1372,32 @@ def test_domain_invitation_created(self):
         self.assertContains(success_page, "Cancel")  # link to cancel invitation
         self.assertTrue(DomainInvitation.objects.filter(email=EMAIL).exists())
 
+    @boto3_mocking.patching
+    def test_domain_invitation_created_for_caps_email(self):
+        """Add user on a nonexistent email with CAPS creates an invitation to lowercase email.
+
+        Adding a non-existent user sends an email as a side-effect, so mock
+        out the boto3 SES email sending here.
+        """
+        # make sure there is no user with this email
+        EMAIL = "mayor@igorville.gov"
```

Comment:
**comment (non-blocking)**: Just a very minor quibble, but all-caps variable names are usually used for module-level constants rather than variables that are just used within this test method.

Reply:
thanks, i will address in this test and related ones before merging